bundles/AdminBundle/Controller/Admin/Document/PageController.php

```diff
@@ -244,12 +244,19 @@ public function displayPreviewImageAction(Request $request)
     public function checkPrettyUrlAction(Request $request)
     {
         $docId = $request->get('id');
-        $path = trim($request->get('path'));
+        $path = (string) trim($request->get('path'));
         $path = rtrim($path, '/');
 
         $success = true;
         $message = null;
 
+        if ($path === '') {
+            return $this->adminJson([
+                'success' => $success,
+                'message' => $message
```

Comment:
null message?

Reply:
I see no side effects in the console, but I can remove it.

Reply:
got it...this is to bypass errors!